.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 16.x, 14.x ]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+
+    - name: Install dependencies
+      run: npm i
```

Comment:
`npm ci` would be a better fit here, since it will install based on the exact versions in `package-lock.json` instead of looking for updated versions based on ranges in `package.json`. It just makes things a little more consistent in CI, which is what we want here 🙂